### PR TITLE
[READY] - init pr-standardize-comment for trigger workflow

### PR DESCRIPTION
**Pull request by sarcasticadmin**
## Description of PR

Fixes: #511

Depends on: #606 

Leverage the opensource workflow: https://github.com/Nebulaworks/action-standardize-pr-comment

Allows us to be able to test the workflows while in the PR vs having to merge and then test (fail on continuous delivery).

## Previous Behavior
- Could only test the trigger GHA workflow on `master`

## New Behavior
- Can test the `trigger` GHA workflow from PR

## Tests
- I believe this will need to be merged in `master` prior to being able to test this for the first time

### CODE REVIEW

File: .github/workflows/trigger.yml [hunk 1]
```diff
@@ -15,12 +15,15 @@ jobs:
       && github.event.issue.pull_request
       && github.actor == 'sarcasticadmin'
     steps:
+    - name: Standardize PR comment
```
Comment (sarcasticadmin) [inline .github/workflows/trigger.yml:18]: /tux openwrt build run-at-test

File: .github/workflows/trigger.yml [hunk 2]
```diff
@@ -15,12 +15,15 @@ jobs:
...
+      id: pr-standardize-comment
+      uses: nebulaworks/action-standardize-pr-comment@v1.0
```
Comment (sarcasticadmin) [inline .github/workflows/trigger.yml:20]: /tux openwrt build run-at-test

**Review by kylerisse — APPROVED** [review]
I believe you are correct @sarcasticadmin , needs to be merged to test. Looks right to me. Worst case something borks and we follow up PR it I suppose...

**Comment by sarcasticadmin** [comment]
Thanks @kylerisse I moved it to `[READY]` and then well see what happens :shipit:

**Pull request merged**